skill-tree /pom.xml

```diff
@@ -52,11 +53,52 @@
 				</exclusion>
 			</exclusions>
 		</dependency>
+
 		<dependency>
 			<groupId>junit</groupId>
 			<artifactId>junit</artifactId>
 			<scope>test</scope>
 		</dependency>
+		<dependency>
+			<groupId>org.springdoc</groupId>
+			<artifactId>springdoc-openapi-starter-webmvc-ui</artifactId>
+			<version>2.1.0</version>
+		</dependency>
+		<dependency>
+			<groupId>org.springframework.security</groupId>
+			<artifactId>spring-security-web</artifactId>
+		</dependency>
+		<dependency>
+			<groupId>org.jetbrains</groupId>
+			<artifactId>annotations</artifactId>
+			<version>17.0.0</version>
+			<scope>compile</scope>
+		</dependency>
+		<dependency>
+			<groupId>io.jsonwebtoken</groupId>
+			<artifactId>jjwt-api</artifactId>
+			<version>0.11.2</version>
```

Comment:
The `io.jsonwebtoken` dependencies seem the share the version. We might benefit from maintaining the version as a property and using the property name here. Something like
`<jsonwebtoken.version>0.11.2</jsonwebtoken.version>`

By doing so, if the version has to be updated later, it'll be a change in a single place and will ensure other dependency versions are also updated and in sync.